webpack.config.js

```diff
@@ -123,7 +127,12 @@ const config = {
 			memo[ name ] = `./${ path }`;
 			return memo;
 		}, {} ),
-		packageNames.reduce( ( memo, packageName ) => {
+		gutenbergPackages.reduce( ( memo, path ) => {
+			const name = camelCaseDash( path );
+			memo[ name ] = `./packages/${ path }`;
```

Comment:
should we add an `src` folder here (and move the files) `./packages/${ path }/src` to match the packages repository setup?

Reply:
I have it locally, need to push. I thought the same when I saw diff on GitHub :)